src/content/structured/components/text-field/code.mdx

```diff
@@ -730,3 +730,168 @@ return (
     helperText="Such as Arabica, Robusta or Liberica"
   />
 </ComponentPreview>
+
+### Full width text field
+
+export const snippetsFullWidth = [
+  {
+    technology: "Web component",
+    snippets: {
+      short: `<ic-text-field  
+  label="What is your favourite coffee?" 
+  required="true" 
+  placeholder="Please enter…" 
+  helper-text="Such as Arabica, Robusta or Liberica" 
+  full-width="true"
+>
+</ic-text-field>`,
```

Comment:
Could this just be moved back onto the previous line, just a minor formatting thing that we do with other examples